tools/upgrade_versions.py

```diff
@@ -26,6 +26,7 @@
 from lib import packages_manager
 from lib import repository
 from lib import rpm_package
+from lib.utils import replace_str_in_file
```

Comment:
This move is not mentioned in the commit message and could be separated to another commit.